frame/benchmarking/src/utils.rs

```diff
@@ -39,13 +50,13 @@ sp_api::decl_runtime_apis! {
 	pub trait Benchmark {
 		/// Dispatch the given benchmark.
 		fn dispatch_benchmark(
-			module: Vec<u8>,
-			extrinsic: Vec<u8>,
+			pallet: Vec<u8>,
```

Comment:
```suggestion
			pallet: &[u8],
```

Shouldn't these also use slice?

Reply:
These values from from `structopt`, so I think they will be Vecs.